indico/htdocs/js/indico/jquery/palettepicker.js

```diff
@@ -111,7 +112,12 @@
                 events: {
                     show: self._updateSelection.bind(self)
                 }
-            });
+            }
+            if (self.options.qtipConstructor) {
+                self.options.qtipConstructor(element, qtipOptions);
```

Comment:
what do you think about using `.call(element, qtipOptions)` and accessing the element as `this` inside the `qtipConstructor`?